scripts/scil_remove_labels.py

```diff
@@ -63,7 +64,7 @@ def main():
 
     # Save final volume
     nii = nib.Nifti1Image(labels_volume, volume_img.affine, volume_img.header)
```

Comment:
side note header is optional (I would prefer header=volume_img.header). We almost never do that in scilpy (providing the header), we usually simply give the affine. Should we start doing it ?
I am @jchoude has some good/bad experience with both.
I know that some output of freesurfer (and DSI Studio) write fancy stuff in the header, but it is usually more pain than joy to deal with them.